kinder/doc/kind-kinder.md

```diff
@@ -0,0 +1,113 @@
+# Kind & Kinder
+
+[Kind](https://github.com/kubernetes-sigs/kind) is awesome!
+
+kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+
+## Target use cases
+
+At high level you should use [Kind](https://github.com/kubernetes-sigs/kind) whenever you want
+a working Kubernetes cluster and _just_ work on it (for DEV, test, CI/CD).
+
+Instead, you should use Kinder when you want to get more control on the process of creating a
+new Kubernetes cluster - from "bare" machines running into a container to a working Kubernetes -.
+
+## Differences
+
+All the [Kind](https://github.com/kubernetes-sigs/kind) commands will be available in kinder,
+side by side with additional commands designed for helping kubeadm contributors.
+
+_Building images:_
+- Kinder support both containerd and as container runtime inside the images
+- Kinder provide support for altering a base/node images and:
+     - Add a Kubernetes version to be used for init (from release, CI/CD or locally build artifacts)
+     - Pre load images tar into the base/node image
+     - Replace the kubectl, kubelet, kubeadm binary to be used for init (from release, CI/CD or locally
+       build artifacts)
+     - Add a Kubernetes version to be used for upgrage (from release, CI/CD or locally build artifacts)
+
+_Creating the cluster:_
+- Kinder support both containerd and as container runtime inside the images
+- Kinder allow to break down the create operation into several atomic actions:
+    - Create machines running into a container
+    - Generate kubeadm config
+    - Generate load balancer config
+
+  NB. Kind manages a unique create operation, that includes create nodes, generate kubeadm/load balancer
+  config init, join (eventually, only the last two operations can be skipped)
+- Kinder allow add an external etcd to the cluster
+- Kinder allow add an external loadbalancer to the cluster also in case if less than 2 control-plane nodes
```

Comment:
allow -> allows
add -> adding
"also in case if less than" -> "even if less than two"